crates/llama-cpp-bindings/src/engine.cc

```diff
@@ -21,7 +21,6 @@ class TextInferenceEngineImpl : public TextInferenceEngine {
   TextInferenceEngineImpl(owned<llama_model> model, owned<llama_context> ctx) :
     model_(std::move(model)),
     ctx_(std::move(ctx)) {
-      batch_ = llama_batch_init(N_BATCH, 0);
```

Comment:
Previous usage of batch api in tabby generates a segmentation fault with updated llama.cpp version - roll back to llama_batch_get_one as workaround, will revisit this when integrating the continuous batching support.